cli/build/local.go

```diff
@@ -32,8 +32,8 @@ func buildProjectLocally(ctx *context.Ctx) error {
 	}
 
 	// tarantoolctl rocks make
-	log.Infof("Running `tarantoolctl rocks make`")
-	rocksMakeCmd := exec.Command("tarantoolctl", "rocks", "make")
+	log.Infof("Running `tarantoolctl rocks make %s`", ctx.Build.Spec)
+	rocksMakeCmd := exec.Command("tarantoolctl", "rocks", "make", ctx.Build.Spec)
```

Comment:
I think that we shouldn't pass found rockspec to `tarantoolctl rocks make` if user didn't specify `--spec` flag.
We can do it simpler:

* `cartridge build` leads to `tarantoolctl rocks make`
* `cartridge build --spec <path>` leads to `tarantoolctl rocks make <path>`